### ISSUE TITLE
VirtualBox under Crouton ... kernel version not there

### ISSUE THREAD

**Issue by CarlFink** [divx118/crouton-packages#72]
Hi. Trying to follow the instructions at https://github.com/dnschneid/crouton/wiki/Build-kernel-headers-and-install-Virtualbox-%28x86%29#installing-kernel-headersimage. Cloned the kernel repository from Github, ran the commands to cycle back through versions to find the exact one in my particular Chromebook's version of ChromeOS. I have a Dell Inspiron 11i 2in1, running ChromeOS 71.0.3578.127 (Official branch). uname -r reports my kernel as 3.18.0-18721-ga0ff0d06b120. The nearest I can get using the commands in the wiki is 3.18.0-18721-gab35955486d6. This would seem to mean I'm completely stuck, unable to create/insmod the VB module.

Any help appreciated.

**Comment by divx118** [comment]
@CarlFink yeah sorry not much I can do about it. This is why I stopped supporting it. Best way to run virtual box is to do it with legacy boot. See https://github.com/divx118/crouton-packages/issues/45#issuecomment-281434222

**Comment by CarlFink** [comment]
Thanks, @divx118.  Unless you feel like doing it, I'll update the wiki linked above to say that this method is no longer supported and not recommended by its author any more, and link to the discussion thread you reference.

**Comment by divx118** [comment]
@CarlFink Great, thanks for doing that.
